survey_form/index.css

```diff
@@ -47,14 +47,61 @@ header::after {
   justify-content: center;
   margin: auto 0;
 }
-.form {
+.form-survey {
   display: flex;
+  flex-direction: column;
   width: 50%;
   margin: 0 auto;
   background-color: #f5f5f5;
   border-radius: 10px;
+  padding-top: 1rem;
 }
 
-.form__tittle {
-  margin: 0 auto;
+.form__survey--title {
+  margin-bottom: 1rem;
+  text-align: center;
+}
+
+.form-survey__input {
+  padding: 6px;
+  margin-left: 30%;
+  margin-right: 30%;
+  margin-bottom: 1rem;
+}
+.form-survey__input:required:invalid {
+  border: 2px solid #f08080;
+  border-radius: 3px;
+}
+
+.form-survey__label {
+  margin-left: 30%;
+  margin-right: 30%;
+  padding-top: 1rem;
```

Comment:
Te dwa paddingi lepiej zastąpić marginami, bo są to odstępy pomiędzy elementami. No i wtedy możesz 4 linie zapisać w jednej ;)